src/components/Sphere.vue

```diff
@@ -0,0 +1,983 @@
+
```

Comment:
Code looks good as it seems to be some advanced module. Issue is causes a million lint errors when building. Might be a good idea to either auto fix the lint issues or put tslint:disable as a comment at beginning of file and tslint:enable at the end of the file